<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>earm_update</class>
 <widget class="QDialog" name="earm_update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#progressBar
{
color: green;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>205</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
color:white;
font-size: 30px;
}</string>
   </property>
   <property name="text">
    <string>Actualización en progreso</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2
{
color:white;
font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Calidad de señal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3
{
color:white;
font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Tiempo de conexión:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_4
{
color:red;
font-size: 30px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_5
{
color:white;
font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Numero de serie:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
